RELEASE.md

```diff
@@ -0,0 +1,109 @@
+<!--
+# Copyright 2018-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Release Notes for 2.34.0
+
+## New Freatures and Improvements
+
+* Python backend supports 
+  [Custom Metrics](https://github.com/triton-inference-server/python_backend/tree/r23.05#custom-metrics) 
+  allowing users to define and report counters and gauges similar to the 
+  [C API](https://github.com/triton-inference-server/server/blob/r23.05/docs/user_guide/metrics.md#custom-metrics).
+
+* Python Triton Client defines the 
+  [Triton Client Plugin API](https://github.com/triton-inference-server/client/tree/r23.05#python-client-plugin-api-beta) 
+  allowing users to register custom plugins to add or modify request headers. 
+  This feature is in beta and is subject to change in future releases.
+
+* Improved performance of model instance creation/removal. When the model 
+  instance group is the only model configuration change, Triton will update the 
+  model with the number of instances needed rather than reloading the model. 
+  This feature is limited to non-sequence models only. Read more about this 
+  feature 
+  [here](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_management.md#modifying-the-model-repository) 
```

Comment:
>https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_management.md#modifying-the-model-repository

Change to `https://github.com/triton-inference-server/server/blob/r23.05docs/user_guide/model_management.md#modifying-the-model-repository`